resources/gtfo.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Looks good, just make sure to add it to `.gitignore` before we add real auth tokens to it.